src/components/SvgMousePosition/component.js

```diff
@@ -0,0 +1,83 @@
+import { Component } from 'react';
+import { subtract } from 'ramda';
+import PropTypes from 'prop-types';
+import { noop } from 'lodash';
+import { throttle } from 'throttle-debounce';
+import { elementInstancePropTypes } from 'models';
+
+class SvgMousePosition extends Component {
+  state = {
+    mousePosition: {},
+    // eslint-disable-next-line react/no-unused-state
+    svgReact: null,
+    hasPosition: false,
+  }
+
+  componentDidMount() {
+    const { svg, delay } = this.props;
+
+    this.isComponentMounted = true;
```

Comment:
Is this needed because of some race condition inside the event listener for `mousemove`?

Reply:
Yes, since I'm using "throttle" to set the component state, there are some scenarios when the component will unmount and after that, I will set the state in an unmounted component.
I don't know if there is a better way to do this 🤔